<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Crossing_Style_Widget</class>
 <widget class="QWidget" name="Crossing_Style_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>122</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="check_handle_length">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Curve</string>
     </property>
     <property name="buddy">
      <cstring>spin_handle_length</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QDoubleSpinBox" name="spin_handle_length">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Size of the curve control handles</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="maximum">
      <double>8192.000000000000000</double>
     </property>
     <property name="value">
      <double>24.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="combo_edge_type"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_edge_type">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="spin_crossing_gap">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Distance between the ends of a thread passing under another thread</string>
     </property>
     <property name="suffix">
      <string>px</string>
     </property>
     <property name="maximum">
      <double>8192.000000000000000</double>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gap</string>
     </property>
     <property name="buddy">
      <cstring>spin_crossing_gap</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="check_crossing_gap"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="check_edge_slide">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Slide</string>
     </property>
     <property name="buddy">
      <cstring>slide_edge_slide</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSlider" name="slide_edge_slide">
     <property name="toolTip">
      <string>Position of the crossing in the edge</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_handle_length</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_handle_length</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_crossing_gap</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_crossing_gap</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_edge_slide</sender>
   <signal>toggled(bool)</signal>
   <receiver>slide_edge_slide</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkbox_toggled()</slot>
 </slots>
</ui>
